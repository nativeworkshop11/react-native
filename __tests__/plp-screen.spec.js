import React from 'react';
import renderer from 'react-test-renderer';
import PlpScreen from '../src/screens/plp-screen';

jest.mock('node-fetch');

jest.mock('@react-navigation/core', () => ({
  useNavigation: jest.fn(),
  useRoute: () => ({params: {keyword: 'bag'}}),
}));

describe('PLP', () => {
  it('should render', () => {
    const wrapper = renderer.create(<PlpScreen />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
