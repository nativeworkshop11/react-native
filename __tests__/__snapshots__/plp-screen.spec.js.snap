// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PLP should render 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Text
    style={
      Object {
        "padding": 20,
      }
    }
  >
    No results found for: bag
  </Text>
</View>
`;
